resources/views/layout-login.twig

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```

Comment:
See comment on app-login.css. We don't actually need this file. Even if we do decide to go with a separate CSS file for one or more pages, that file can easily be incorporated into `layout.twig` with if statements, rather than duplicating the main layout file.

What makes more sense though, is if the login page needs additional css, we can add css directly to the page (or call additional files) in the header styles block defined in the layout file. That block just needs to be added to the login template.

```twig
{% block headerStyles %}
    We can call any additional CSS files or add in custom CSS here on the login template itself.
{%endblock%}
```